### DISCUSSION TITLE
Ctrl + H Keybind does not work while in tool mode

### DISCUSSION THREAD

**Discussion by Hatchibombotar** [Mojang/minecraft-editor#9] [Bugs]
Reproduction steps:

1. Go to File > Pause Screen > Settings > Creator
2. Enable content log GUI
3. Go back into game
4. Make sure you are in editor mode
5. Use Ctrl + H keybind

Expected result: Content Log opens (as in crosshair mode)
Actual result: Nothing happens.

**Comment by minecraftchloe** [comment]
Thank you! We'll triage this. One thing we are considering is whether or not this keybinding (Ctrl + H) will ever want to be used in Tool Mode for an extension. Benefit to having Tool and Crosshair is that Crosshair inherits Vanilla bindings, whereas Tool is a "blank slate" to some extent.

**Comment by minecraftchloe** [comment]
Alright, triaged with the team: For now, we're going to keep Ctrl + H content log history in Crosshair, but never fear -- we have plans to implement a notifications/logging system in Tool. Marking this one as `wontfix`!
